app/templates/Membership_Application_Form.html.twig

```diff
@@ -245,6 +245,7 @@
 
 					<div class="container clearfix">
 
+{% if 'PPL' in paymentTypes %}
```

Comment:
Does this exploit that `BEZ` is still the default value if no `payment_type` is received via the form? Sure is a quick way to get this back to state it was before the introduction of `PPL`, but looks confusing (disabling BEZ if PPL is not allowed)...